app/models/miq_report.rb

```diff
@@ -223,4 +225,20 @@ def remove_loading_relations_for_virtual_custom_attributes
     vc_attributes = CustomAttributeMixin.select_virtual_custom_attributes(cols).present?
     include.delete(:custom_attributes) if vc_attributes.present? && include && include[:custom_attributes].blank?
   end
+
```

Comment:
@lpichler Please add a comment here indicating that these methods are for supporting legacy reports that may have some of the columns that are being removed by this PR in them.